service/internal/changedetection/components.go

```diff
@@ -0,0 +1,38 @@
+package changedetection
+
+import (
+	releasev1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1"
+)
+
+//components which matter to trigger update
+// Kubernetes, aws-cni, ...
+
+var components = []string{"kubernetes", "etcd", "aws-cni", "calico", "containerlinux"}
```

Comment:
Currently these are the components we want to check if they have changed, a new version of those will update CF stack.